unfetter-discover/unfetter-ui
# Issue 733/edit assessment

fixes unfetter-discover/unfetter#735
fixes unfetter-discover/unfetter#733

#To Test
* Pull this PR
* Visit the risk assessments results page
* Edit an indicator ie assesed object on the far right (click the pencil and select a new value)
* Verify the risk calculation update across the page
* Verify the middle column ranks the attack patterns by risk
* `npm run test` verify no errors
